applications/StructuralMechanicsApplication/custom_python/add_custom_processes_to_python.cpp

```diff
@@ -79,6 +80,12 @@ void  AddCustomProcessesToPython(pybind11::module& m)
         .def(py::init< ModelPart&, Parameters >())
         ;
 
+    py::class_<AssignNodalElementsToNodesProcess, AssignNodalElementsToNodesProcess::Pointer, Process>(m, "AssignNodalElementsToNodesProcess")
+        .def(py::init<Model&, Parameters>())
+        .def(py::init<ModelPart&>())
+        .def(py::init<ModelPart&, Parameters >())
```

Comment:
Aiming at standarization, I'd only implement and expose the `Model` - `Parameters` ctor for new features.